Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -79,16 +79,20 @@ private enum CardinalAxisType
         }
 
         /// <summary>
-        /// This enum is used internally to define how an object's bounds are calculated in order to fit the boundingbox
+        /// This enum defines what volume type the bound calculation depends on and its priority
         /// to it.
         /// </summary>
-        private enum BoundsCalculationMethod
+        public enum BoundsCalculationMethod
```

Comment:
It's not immediately clear to me what these enums mean, which makes me think folks consuming this will be confused as well. Could you please explain what each value means in the summary comment? This way it will be captured in the documentation so folks can reference it.